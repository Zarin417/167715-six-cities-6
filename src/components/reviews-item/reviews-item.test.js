import React from "react";
import {render} from "@testing-library/react";
import ReviewsItem from "./reviews-item";

it(`ReviewsItem should render correctly`, () => {
  const review = {
    offerId: 2,
    comment: `Cozy`,
    date: `2019-12-08T14:13:56.569Z`,
    id: 3,
    rating: 3,
    user: {
      avatarUrl: `img/avatar-max.jpg`,
      id: 6,
      isPro: true,
      name: `Frederik`
    }
  };

  const {container} = render(
      <ReviewsItem review={review} />
  );

  expect(container).toMatchSnapshot();
});
