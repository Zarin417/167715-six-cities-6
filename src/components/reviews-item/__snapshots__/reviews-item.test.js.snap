// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReviewsItem should render correctly 1`] = `
<div>
  <li
    class="reviews__item"
  >
    <div
      class="reviews__user user"
    >
      <div
        class="reviews__avatar-wrapper user__avatar-wrapper"
      >
        <img
          alt="Reviews avatar"
          class="reviews__avatar user__avatar"
          height="54"
          src="img/avatar-max.jpg"
          width="54"
        />
      </div>
      <span
        class="reviews__user-name"
      >
        Frederik
      </span>
      <span
        class="property__user-status"
      >
        Pro
      </span>
    </div>
    <div
      class="reviews__info"
    >
      <div
        class="reviews__rating rating"
      >
        <div
          class="reviews__stars rating__stars"
        >
          <span
            style="width: 60%;"
          />
          <span
            class="visually-hidden"
          >
            Rating
          </span>
        </div>
      </div>
      <p
        class="reviews__text"
      >
        Cozy
      </p>
      <time
        class="reviews__time"
        datetime="2019-12-08"
      >
        December 8
      </time>
    </div>
  </li>
</div>
`;
